src/trx/game/shell/flow.c

```diff
@@ -255,6 +251,12 @@ int32_t Shell_Main(const SHELL_ARGS *args)
     ASSERT(args != nullptr);
     LOG_INFO("Game directory: %s", File_GetGameDirectory());
 
+    // Set the g_TRVersion early. Later on it will be overriden by the
+    // level loader, but this lets the game load good config defaults.
+    g_TRVersion = args->engine_version != 0 ? args->engine_version
```

Comment:
This should be fine because `args->mod` is already known at this time thanks to `int main` doing `Shell_ScanAvailableMods` and `Shell_ParseArgs`, which assigns the `mod`.